actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -390,11 +393,19 @@ def preload_link_tag(source, options = {})
       #   # => <img src="/rails/active_storage/representations/.../tiger.jpg" />
       #   image_tag(user.avatar.variant(resize_to_limit: [100, 100]), size: '100')
       #   # => <img width="100" height="100" src="/rails/active_storage/representations/.../tiger.jpg" />
+      #
+      # If automatic resizing is enabled, the last example can be simplified:
+      #
+      #   image_tag(user.avatar, size: 100)
+      #   # => <img width="100" height="100" src="/rails/active_storage/representations/.../tiger.jpg"
```

Comment:
I've added the documentation here and a bit above. I'm not sure if this is what you meant though.